dbms/src/Common/StringUtils/StringUtils.h

```diff
@@ -149,3 +149,99 @@ inline bool equalsCaseInsensitive(char a, char b)
 {
     return a == b || (isAlphaASCII(a) && alternateCaseIfAlphaASCII(a) == b);
 }
+
+inline bool isPunctuation(char c)
```

Comment:
We already have this function in https://github.com/pingcap/tiflash/blob/9e35992d43c524a8ddf4f5efd0096bdb6ee1c2b7/dbms/src/Common/MyTime.cpp#L260

Reply:
removed